We'll take care of stripping thank you very much

--- config.mk.in
+++ config.mk.in
@@ -87,8 +87,8 @@
 k		= $(KPREFIX)
 
 INSTALLSCRIPT	= $(INSTALL) -m 755
-INSTALLBIN	= $(INSTALL) -m 755 -s
-INSTALLSUID	= $(INSTALL) -m 4755 -s
+INSTALLBIN	= $(INSTALL) -m 755
+INSTALLSUID	= $(INSTALL) -m 4711
 INSTALLMOD	= $(INSTALL) -m 600
 INSTALLMAN	= $(INSTALL) -m 644
 MAN1DIR		= $(MANDIR)/man$(MAN1EXT)
